web/content/docs/benchmarks/creep-after-excavation-bgra/CreepAfterExcavation.md

```diff
@@ -53,7 +53,7 @@ assume a steady state of stress and temperature of excavation at the
 beginning of the current simulation. For this assumption, the boundary
 conditions are given as:
 
--   top boundary: $\tau_x = \sigma_x=0$, $\tau_y=\sigma_y=70$ MPa,
+-   top boundary: $\tau_x =$$\sigma_x$$_y=0$, $\tau_y=\sigma_y=-10$ MPa,
```

Comment:
Not sure about double $$ here, though the rendering looks ok. Maybe `$sigma_{xy} = 0$`....

Reply:
$sigma_{xy} = 0$ did not work (actually, {} in subscript does not work  occasionally). That's why I split it into three formulas as  `$\tau_x =$$\sigma_x$$_y=0$`. With the splitting, you can see that $$ are not doubled $ for one formulas.

By the way, the formulas in the benchmark documentation have already been corrected according to the comments on the report by @nagelt.